engine-tests/src/main/java/org/terasology/Environment.java

```diff
@@ -34,12 +36,13 @@
 
     private static final Logger logger = LoggerFactory.getLogger(Environment.class);
 
+    protected Context context;
```

Comment:
I had it first private too, but the field gets used a lot in the sub class HeadlessEnvironment and I think:
context.put(...)
reads better than:
getContext().put(...)